tests/unit/components/cylc/task.vue.spec.js

```diff
@@ -7,47 +7,14 @@ import Task from '@/components/cylc/Task'
 
 describe('Task component', () => {
   it('should initialize props', () => {
-    const wrapper = shallowMount(Task, {
-      propsData: {
-        status: '',
-        isHeld: true
+    const wrapper = mount(Task, {
+      context: {
+        props: {
+          status: '',
+          isHeld: true
+        }
       }
     })
-    expect(wrapper.props().status).to.equal('')
-    expect(wrapper.props().isHeld).to.equal(true)
-    expect(wrapper.props().progress).to.equal(0)
-  })
-  it('should compute progress style', () => {
-    // local this gives the values used in the computed function, as we are not mounting the component
-    const localThis = {
-      progress: 25
-    }
-    expect(Task.computed.progressStyle.call(localThis)['stroke-dashoffset']).to.equal(157 - (0.25 * 157))
```

Comment:
Why have these tests been removed?

Reply:
Ah, yup. We cannot have `computed` values in functional components. A functional component is stateless, so it has no `data`, so nothing to compute. The code that was `computed` is still "computed", but it's done during the component creation, inlined with the backquotes. They are "computed" from the `props` values.